argocd/overlays/moc-infra/projects/peribolos-service.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: argoproj.io/v1alpha1
+kind: AppProject
+metadata:
+  name: peribolos-as-a-service
```

Comment:
Filename should match this name here, I think. `peribolos-service.yaml` file vs `peribolos-as-a-service` project.